godot-git-plugin/src/git_common.h

```diff
@@ -5,10 +5,23 @@
 
 #include <git2.h>
 
-void check_git2_errors(int error, const char *message, const char *extra);
+struct credentials {
```

Comment:
It should be `Credentials` :) Also I think this would leak memory, it maybe I am misunderstanding how this has been used above